recipe/meta.yaml

```diff
@@ -7,25 +7,27 @@ package:
 source:
   # The source url changed for  0.37.*
   url: https://pypi.io/packages/source/S/SQLAlchemy-Utils/SQLAlchemy-Utils-{{ version }}.tar.gz
-  sha256: a6aaee154f798be4e479af0ceffaa5034d35fcf6f40707c0947d21bde64e05e5
+  sha256: a2181bff01eeb84479e38571d2c0718eb52042f9afd8c194d0d02877e84b7d74
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  # Intervals is not available on s390x
+  skip: true  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
   host:
     - python
     - pip
     - wheel
     - setuptools
+    - sqlalchemy 1.4.39
+    - importlib_metadata 6.0.0 # [py<38]
   run:
     - python
     - six
     - sqlalchemy >=1.0
```

Comment:
It requires `sqlalchemy >=1.3`, see https://github.com/kvesteri/sqlalchemy-utils/blob/fd94d17621b8e741b8c7353b700c19a4f38c7015/setup.py#L78

Reply:
Thank you @skupr-anaconda